### PR TITLE
Release 2.0.2 (develop)

### PR DESCRIPTION
**Pull request by arebra**
Version 2.0.2

* Various bug fixes
* Various corrections and typo fixes
* Corrected information about 3D-Secure
* Added paragraphs about different consumer flows in Checkin
* Added link to gift card on the front page

### CODE REVIEW

**Review by NullableInt — APPROVED** [review]

File: resources/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ description: |
...
 
+
+## 01 Dec 2020
+
+### Version 2.0.2
+
+* Various bug fixes
+* Various typo fixes
+* Corrected information about [3D-Secure][card]
+* Added sections to [Checkin][Checkin]
```
Comment (asbjornu) [inline resources/release-notes.md:21]: Which sections were added to Checkin?

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by asbjornu** [comment]
Now that #788 is merged, perhaps we can fold that into this release as well?

**Comment by arebra** [comment]
Sure! How do I do that? :)

File: resources/release-notes.md (hunk 1 above)
Comment (arebra) [reply resources/release-notes.md]: Better now? :)

**Review by asbjornu — APPROVED** [review]
👏

**Pull request merged**